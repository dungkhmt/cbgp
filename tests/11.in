30 30
30 43
17 12
12 22
22 24
24 9
9 4
4 23
23 20
20 17
2 28
28 14
14 8
8 15
15 3
3 19
19 5
5 30
30 2
27 13
13 25
25 6
6 27
11 16
16 26
26 7
7 18
18 11
10 21
21 29
29 1
1 10
28 12
27 2
8 4
7 1
19 20
22 9
30 17
10 11
25 15
14 13
5 6
23 3
29 16
1
15 3 21 11 7 1 0 0 30 2 12 2 27 3 7 3 16 16 17 5 19 7 19 15 12 4 10 6 8 2 23 4 30 4 21 9 31 1 34 0 19 5 18 16 6 0 17 17 10 2 25 4 13 3 18 14 20 4 29 3 
