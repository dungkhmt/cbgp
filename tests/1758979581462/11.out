(16, 11), (11, 21), (21, 23), (23, 8), (8, 3), (3, 22), (22, 19), (19, 16), (1, 27), (27, 13), (13, 7), (7, 14), (14, 2), (2, 18), (18, 4), (4, 29), (29, 1), (26, 12), (12, 24), (24, 5), (5, 26), (10, 15), (15, 25), (25, 6), (6, 17), (17, 10), (9, 20), (20, 28), (28, 0), (0, 9), (27, 11), (26, 1), (7, 3), (6, 0), (18, 19), (21, 8), (29, 16), (9, 10), (24, 14), (13, 12), (4, 5), (22, 2), (28, 15), 
numberIntersection=0 
minAngle=0.6164554662469941 
Var(edgeLength)=529.3157320097831 
minDistance=0.06643638388299197 
Var(nodePos)=1352.7777777777778 
sumAngle=27.063717153948613 
sumDistance=46972.4599141988 
minEdgeLength=1.0 
sumEdgeLength=956.6602563374541 

0.0000000000 0.6164554662 529.3157320098 0.0664363839 1352.7777777778 27.0637171539 46972.4599141988 1.0000000000 956.6602563375

0: (24, 3),
1: (29, 64),
2: (3, 1),
3: (0, 3),
4: (84, 2),
5: (13, 2),
6: (58, 7),
7: (13, 10),
8: (29, 67),
9: (25, 44),
10: (31, 59),
11: (29, 65),
12: (15, 8),
13: (14, 10),
14: (14, 9),
15: (46, 10),
16: (89, 2),
17: (33, 59),
18: (85, 1),
19: (86, 0),
20: (44, 16),
21: (30, 65),
22: (2, 0),
23: (31, 66),
24: (14, 8),
25: (47, 13),
26: (17, 5),
27: (28, 64),
28: (38, 16),
29: (83, 3),

Time taken: 46.737 s
24 3 29 64 3 1 0 3 84 2 13 2 58 7 13 10 29 67 25 44 31 59 29 65 15 8 14 10 14 9 46 10 89 2 33 59 85 1 86 0 44 16 30 65 2 0 31 66 14 8 47 13 17 5 28 64 38 16 83 3 
0: (24, 3), 1: (29, 64), 2: (3, 1), 3: (0, 3), 4: (84, 2), 5: (13, 2), 6: (58, 7), 7: (13, 10), 8: (29, 67), 9: (25, 44), 10: (31, 59), 11: (29, 65), 12: (15, 8), 13: (14, 10), 14: (14, 9), 15: (46, 10), 16: (89, 2), 17: (33, 59), 18: (85, 1), 19: (86, 0), 20: (44, 16), 21: (30, 65), 22: (2, 0), 23: (31, 66), 24: (14, 8), 25: (47, 13), 26: (17, 5), 27: (28, 64), 28: (38, 16), 29: (83, 3), 
numberIntersection=0 minAngle=0.6164554662469941 sumAngle=27.063717153942566 minEdgeLength=1.0 sumEdgeLength=956.6602563374543 minDistance=0.06643638388299197 sumDistance=46972.45991419929 
